bazel/repositories.bzl

```diff
@@ -564,11 +564,11 @@ cc_library(
     _maybe(
         http_archive,
         name = "io_bazel_rules_go",
-        sha256 = "9d7d865b92cc1e16e7c1d53b3d0f6532252a512d7e09d6f074fe41423f036cca",
-        strip_prefix = "rules_go-0.17.1",
         urls = [
-            "https://github.com/bazelbuild/rules_go/archive/0.17.1.tar.gz",
+            "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/0.18.5/rules_go-0.18.5.tar.gz",
+            "https://github.com/bazelbuild/rules_go/releases/download/0.18.5/rules_go-0.18.5.tar.gz",
         ],
+        sha256 = "a82a352bffae6bee4e95f68a8d80a70e87f42c4741e6a448bec11998fcc82329",
```

Comment:
Please run buildifier to keep the ordering stable.

Reply:
Done.